nixos/modules/hardware/ledger.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.ledger;
+
+  udevRules = pkgs.stdenv.mkDerivation {
+    name = "ledger-udev-rules";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "LedgerHQ";
+      repo = "udev-rules";
+      rev = "20cc1651eb551c4855aaa56628c77eaeb3031c22";
+      sha256 = "0riydkc4in10pv4qlrvbg3w78qsvxly5caa3zwyqcmsm5fmprqky";
+    };
+
+    configurePhase = false;
+    buildPhase = false;
+
+    installPhase = ''
+      mkdir -p $out/lib/udev/rules.d
+      cp 20-hw1.rules $out/lib/udev/rules.d/20-ledger.rules
+    '';
+
+    meta = {
+      description = "udev rules for Ledger devices";
```

Comment:
This is missing the `homepage` attribute that should be the value of the github repo.